### PR TITLE
Package picasso.0.4.0

### PR DESCRIPTION
**Pull request by ghilesZ**
### `picasso.0.4.0`
Abstract elements drawing library
A toolbox for drawing abstract elements

---
* Homepage: https://github.com/ghilesZ/picasso
* Source repo: git+https://github.com/ghilesZ/picasso
* Bug tracker: https://github.com/ghilesZ/picasso/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/picasso/picasso.0.4.0/opam [hunk 1]
```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Ghiles Ziat <ghiles.ziat@epita.fr"
+authors: [
+  "Ghiles Ziat <ghiles.ziat@epita.fr"
+  "Matthieu Journault <Matthieu.journault@lip6.fr>"
+  ]
+homepage: "https://github.com/ghilesZ/picasso"
+bug-reports: "https://github.com/ghilesZ/picasso/issues"
+dev-repo: "git+https://github.com/ghilesZ/picasso"
+build: [
```
Comment (mseri) [inline packages/picasso/picasso.0.4.0/opam:10]: Can you add a `license` field? You can look in https://spdx.org/licenses/ for the indentifiers to use

File: packages/picasso/picasso.0.4.0/opam [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/picasso/picasso.0.4.0/opam:11]:
```suggestion
  ["dune" "subst"] {dev}
```
That is from a bug in dune < 2.7, we can fix it manually here on the PRs, no action needed from your side

**Comment by mseri** [comment]
Is the following failure related to changes in this library?
```
#=== ERROR while compiling absolute.0.1 =======================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.14.1 | file:///home/opam/opam-repository
# path                 ~/.opam/4.14/.opam-switch/build/absolute.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p absolute -j 31
# exit-code            1
# env-file             ~/.opam/log/absolute-7-73b5f4.env
# output-file          ~/.opam/log/absolute-7-73b5f4.out
### output ###
# (cd _build/default && /home/opam/.opam/4.14/bin/ocamlyacc src/parser/modParser.mly)
# 2 rules never reduced
# (cd _build/default && /home/opam/.opam/4.14/bin/ocamlyacc src/parser/parser.mly)
# 2 rules never reduced
# (cd _build/default && /home/opam/.opam/4.14/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.libabsolute.objs/byte -I /home/opam/.opam/4.14/lib/apron -I /home/opam/.opam/4.14/lib/apronext -I /home/opam/.opam/4.14/lib/gmp -I /home/opam/.opam/4.14/lib/picasso -no-alias-deps -o src/.libabsolute.objs/byte/out.cmo -c -impl src/print/out.ml)
# File "src/print/out.ml", line 66, characters 35-41:
# 66 |       if !tex then to_latex render "name";
#                                         ^^^^^^
# Error: The function applied to this argument has type
#          ?filename:string -> ?tikz_only:bool -> unit
# This argument cannot be applied without label
```

Let me know and I will add the necessary upper bounds

File: packages/picasso/picasso.0.4.0/opam (hunk 1 above)
Comment (mseri) [reply packages/picasso/picasso.0.4.0/opam:10]: Please let me know the license and we can update the PR and merge

**Comment by mseri** [comment]
Thanks

File: packages/picasso/picasso.0.4.0/opam (hunk 1 above)
Comment (mseri) [reply packages/picasso/picasso.0.4.0/opam:10]: Ping @ghilesZ, if you let me know what license to use I can update the opam file and merge the PR

Comment (ghilesZ) [reply packages/picasso/picasso.0.4.0/opam:10]: Sry for the delay, anything will do, let's say LGPL

Comment (mseri) [inline packages/picasso/picasso.0.4.0/opam:10]:
```suggestion
license: "LGPL-2.0-or-later"
build: [
```

Comment (mseri) [reply packages/picasso/picasso.0.4.0/opam:10]: Thanks!

**Comment by mseri** [comment]
Thanks!

**Comment by mseri** [comment]
Feel free to announce the release on discuss.ocaml.org. We have a community category and an announcement tag for this purpose

**Pull request merged**